### PR TITLE
Persistent history

### LINKED ISSUE #54: Kete: Add support for persistent history
See https://docs.python.org/3/library/readline.html#history-file

### PR DESCRIPTION
**Pull request by bentiss**
Fixes #54, provide a fix for DBus introspection and rename `tuhi-kete.py` into `tuhi_kete.py`, because python doesn't like minuses in the name if you intend to import the file as a module

### CODE REVIEW

File: tools/tuhi_kete.py [hunk 1]
```diff
@@ -562,6 +566,20 @@ def __init__(self, completekey='tab', stdin=None, stdout=None):
         # patching get_names to hide some functions we do not want in the help
         self.get_names = self._filtered_get_names
 
+        try:
+            os.mkdir(CONFIG_PATH)
+        except FileExistsError:
+            pass
+
+        self._history_file = os.path.join(CONFIG_PATH, 'history')
```
Comment (whot) [inline tools/tuhi_kete.py:574]: I pushed this but in half-hindsight I wonder if it should be `histfile` just for googleability. Not that this really matters...

File: org.freedesktop.tuhi.json [hunk 2]
```diff
@@ -127,7 +127,7 @@
                 "python3 setup.py install --verbose"
             ],
             "post-install": [
-                "cp tools/tuhi-kete.py /app/lib/python3.6/site-packages/tuhi_kete.py",
+                "cp tools/tuhi_kete.py /app/lib/python3.6/site-packages/tuhi_kete.py",
```
Comment (whot) [inline org.freedesktop.tuhi.json:130]: Any reason we can't just name this thing `kete.py` and only name it `tuhi-kete`  on install? Seems better than messing around with underscores.

**Review by whot — CHANGES_REQUESTED** [review]
I pushed the first and third patch directly to master (after fixing a typo). For patch 2 I think we should just use `kete.py`.

Also "doesn't like really much" → "doesn't really like" in the commit message please.

File: org.freedesktop.tuhi.json (hunk 2 above)
Comment (bentiss) [reply org.freedesktop.tuhi.json:130]: OK, I'll amend this in #70

File: tools/tuhi_kete.py (hunk 1 above)
Comment (bentiss) [reply tools/tuhi_kete.py:574]: OK, better fixing that right now. I'll fix it and push to master.

**Comment by bentiss** [comment]
Closing, as the only remaining patch is also in #70

File: tools/tuhi_kete.py (hunk 1 above)
Comment (bentiss) [reply tools/tuhi_kete.py:574]: FWIW:
   e63785b..c86c318  master -> master

**Pull request closed without merging**